### PR TITLE
gulp-cli 2.3.0 (new formula)

### PR DESCRIPTION
**Pull request by SeekingMeaning**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Review by dawidd6 — APPROVED** [review]

File: Formula/gulp-cli.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  depends_on "node"
+
+  def install
+    system "npm", "install", *Language::Node.std_npm_install_args(libexec)
+    bin.install_symlink Dir["#{libexec}/bin/*"]
+  end
+
+  test do
+    system "npm", "init", "-y"
+    system "npm", "install", *Language::Node.local_npm_install_args, "gulp"
```
Comment (SMillerDev) [inline Formula/gulp-cli.rb:19]: Why does it need to install gulp again?

Comment (SeekingMeaning) [reply Formula/gulp-cli.rb:19]: Found a good explanation: https://stackoverflow.com/a/35573218

Comment (SMillerDev) [reply Formula/gulp-cli.rb:19]: Okay, that's a weird system but sure.

**Review by SMillerDev — APPROVED** [review]

**Review by chenrui333 — APPROVED** [review]
💯 🚀

**Pull request closed without merging**